tox.ini

```diff
@@ -133,7 +133,9 @@ commands_pre=
     pip install -r {envtmpdir}/requirements.txt
     pip freeze
 commands=
-    pytest astropy/astropy/io/misc/asdf --open-files --run-slow --remote-data -W "ignore:The property AsdfFile.blocks has been deprecated:asdf.exceptions.AsdfDeprecationWarning:asdf.asdf:661"
+    pytest astropy/astropy/io/misc/asdf --open-files --run-slow --remote-data \
+        -W "ignore:The property AsdfFile.blocks has been deprecated:asdf.exceptions.AsdfDeprecationWarning:asdf.asdf:661" \
```

Comment:
This was fixed by astropy/astropy#14358 so it is unnecessary.

Reply:
I removed this line.